app/views/shared/_webchat_popup.html.erb

```diff
@@ -9,18 +9,20 @@
     </svg>
   </div>
   <div class="mobile-webchat__form">
-    <h4><%= t('contact_panels.chat_popup.title') %></h4>
+    <% if pensions_and_retirement_page? %>
+      <%= render 'shared/webchat_popup_pensions' %>
+    <% else %>
+    <h4><b><%= t('contact_panels.chat_popup.title') %><b></h4>
```

Comment:
Do we need the `<b>` tag here? Doesn't seem to make much difference if it's removed. However, if we do, the end tag is wrong (`<b>` not `</b>`).

Reply:
I've removed this tag now and added in some styles for the bold title text.